roles/openshift-management/tasks/prune-projects.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Set username fact (if this is run by an Ansible Tower job)
+  set_fact:
+    run_as: "--as='{{ tower_user_name }}'"
+  when: 
+  - tower_user_name is defined
+  - tower_user_name|trim != ''
+
+- name: Get list of projects
+  shell: oc {{ run_as | default('') }} get project --no-headers | awk '{print $1}'
+  environment:
+    KUBECONFIG: "{{ kubeconfig }}"
+  register: projects 
+
+- name: Get OpenShift project timestamps
+  shell: oc {{ run_as | default('') }} get project {{ item }} --template={{ '{{' }}.metadata.creationTimestamp{{ '}}' }}
+  environment:
+    KUBECONFIG: "{{ kubeconfig }}"
+  register: project_timestamp
+  with_items: "{{ projects.stdout_lines }}"
```

Comment:
@sabre1041 hmm ... it shouldn't even hit that area if pruning projects are disabled. I just tried it in my Ansible 2.2.1.0 env, and I don't see this problem. Were you using 2.2.0.0?